### PR TITLE
feat: topics and reenable v2 docs

### PR DESCRIPTION
**Pull request by chronark**

### CODE REVIEW

**Comment by sancar** [comment]
Since v1 is deprecated now, I think we should update all the /v1 examples from other parts of the doc. 
An example: 
The curl code example in https://upstash.com/docs/qstash/overall/getstarted#publish-a-message
Wdyt ?

**Comment by sancar** [comment]
https://upstash.com/docs/qstash/features/delay 
Delay page will need an update for v2. `Upstash-Not-Before` is not supported in v2 schedules.

**Comment by sancar** [comment]
https://upstash.com/docs/qstash/howto/delete-schedule
There is a `TODO: Fahreddin` . Not related the pr, this is even in the live one. 
The following link in this page seems wrong  -> [schedules api](https://docs.upstash.com/qstash/openapi#tag/Schedules)

**Comment by sancar** [comment]
https://upstash.com/docs/qstash/howto/debug-logs 
This page needs to be adopted to v2 as we don't have exact same events as v1.

**Comment by sancar** [comment]
Dead letter queues page is not listed in the left part under `Features`. 
I have accessed it from the link in the migration page.

**Comment by sancar** [comment]
A typo in http://localhost:3000/qstash/api/messages/create
ContentType -> Content-Type

Copy past error on the response part of the same page 
http://localhost:3000/qstash/api/messages/create#response
publish does not return `scheduleId`

File: qstash/api/dlq/deleteMessage.mdx [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+## Request
+
+<ParamField path="dlqId" type="string">
+  The dlq id of the message you want to remove. You will see this id when listing all messages in the dlq with the [/v2/dlq/messages](/qstash/api/dlq/listMessages) endpoint.
+</ParamField>
+
+## Response
+
+The endpoint doesn't return anything, a status code of 200 means the message is removed from the DLQ.
```
Comment (sancar) [inline qstash/api/dlq/deleteMessage.mdx:18]: Should we mention the failure/not found case ?

File: qstash/api/topics/add-endpoint.mdx [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+<ParamField path="topicName" type="string" required>
+  The name of your topic. If it doesn't exist yet, it will be created.
+</ParamField>
+
+<ParamField body="endpoints" type="Array" required>
+  The endpoints to add to the topic.
+
+<Expandable defaultOpen>
+
+  <ParamField body="name" type="string" required >
```
Comment (sancar) [inline qstash/api/topics/add-endpoint.mdx:21]: name is optional

File: qstash/api/topics/remove-endpoint.mdx [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+authMethod: "bearer"
+---
+
+Remove one or multiple endpoints from a topic. If all endpoints have been removed, the topic will be deleted.
+
+
+## Request
+
+<ParamField path="topicName" type="string" required>
+  The name of your topic. If it doesn't exist yet, it will be created.
```
Comment (sancar) [inline qstash/api/topics/remove-endpoint.mdx:14]: Here if the topic does not exist, we return error.
```
 errors.NotFoundError{ResourceType: "topic", ResourceId: topicName}
 ```

Comment (chronark) [reply qstash/api/topics/remove-endpoint.mdx:14]: sorry, copy paste mistake

**Pull request merged**